scripts/generate_abi.py

```diff
@@ -0,0 +1,97 @@
+import json
+import tempfile
+from pathlib import Path
+from typing import Any
+
+import click
+from psutil import Popen
+
+COMPILE_COMMAND = """
+git clone git@github.com:beamer-bridge/beamer.git {beamer_path} &&
+cd {beamer_path} &&
+git fetch &&
+git checkout {commit_id} &&
+$(which ape) compile
+"""
+
+
+def _get_contract_commit_ids(deployment_path: Path) -> dict[str, set[str]]:
+    file_contract_commits: dict[str, set[str]] = {}
+    for file_path in deployment_path.glob("*.json"):
+        with open(file_path, "r") as file:
```

Comment:
Please use a name other than `file`, which is a builtin Python type. Commonly used names are `f`, `fp` etc.